profile/README.md

```diff
@@ -1,6 +1,7 @@
 # Siemens ❤ Open Source
 
-Siemens uses, maintains and contributes to Open Source.
+Siemens is a [technology company](https://www.siemens.com/global/en/company/about.html) that uses, maintains and contributes to Open Source.
```

Comment:
Thanks @fgreinacher! Would it make sense to expand that a bit so it doesn't sound like that's the only thing we do (even though I'd love that :slightly_smiling_face: )? Maybe like this one straight from our [YT channel](https://www.youtube.com/@siemens) if it's not already outdated:

```suggestion
Siemens is a [technology company](https://www.siemens.com/global/en/company/about.html) focused on industry, infrastructure, transport, and healthcare. We use, maintain and contribute to Open Source.
```

If that's too clunky we can just go ahead with this one though. Just an idea!

Reply:
Yeah, that's a good point @nejch!

Should we add a line break to keep it a bit easier to read?

```suggestion
Siemens is a [technology company](https://www.siemens.com/global/en/company/about.html) focused on industry, infrastructure, transport, and healthcare.

We use, maintain and contribute to Open Source.
```

Reply:
Ah yes, that probably looks less cluttered :bow: